### DISCUSSION TITLE
sidebar/sanfene/spring

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#108] [Announcements]
# sidebar/sanfene/spring

图文详解 35 道Spring面试高频题，这次吊打面试官，我觉得稳了（手动 dog）

https://tobebetterjavaer.com/sidebar/sanfene/spring.html

**Comment by xuliji** [comment]
二哥好，spring七种事务传播规则少了一个NEVER

**Comment by Kddk-K** [comment]
9.26一刷

**Comment by hachinekooo** [comment]
1.11一刷

**Comment by hachinekooo** [comment]
2.11
